.github/workflows/clang.yml

```
@@ -53,28 +52,33 @@ jobs:
+      - name: 'Raise Priority for apt.postgresql.org'
+        run: |
+          cat << EOF >> ./pgdg.pref
+          Package: *
+          Pin: release o=apt.postgresql.org
+          Pin-Priority: 600
+          EOF
+          sudo mv ./pgdg.pref /etc/apt/preferences.d/
+          sudo apt update
 
       - name: Add PostgreSQL APT repository
         run: |
```

actionlint (1.7.7)

property "psql" is not defined in object type {release: string}

(expression)